blobstore/src/test/java/org/jclouds/blobstore/integration/internal/BaseContainerLiveTest.java

```diff
@@ -112,12 +116,46 @@ public void testPublicAccessInNonDefaultLocationWithBigBlob() throws Interrupted
       runCreateContainerInLocation(payload, nonDefault);
    }
 
+   @Test(groups = "live", dependsOnMethods = "testPublicAccess")
+   public void testDelimiterList() throws InterruptedException {
+      final String containerName = getScratchContainerName();
```

Comment:
Do you actually want `getScratchContainerName` or simply `getContainerName`?  The latter uses the recycled containers instead of allocating a new one.